workspaces/ui-v2/src/utils/pathMatcher.ts

```diff
@@ -20,12 +20,14 @@ export function urlStringToPathComponents(
   return components;
 }
 
-export const makePattern = (components: PathComponentAuthoring[]) => {
+export const createPathFromPathComponents = (
+  components: PathComponentAuthoring[]
+) => {
   return (
     '/' +
     components
       .map((i) => {
-        return i.isParameter ? `:${i.name}` : i.originalName;
+        return i.isParameter ? `{${i.name}}` : i.originalName;
```

Comment:
Previously, this was piped into the `path-to-regexp` lib, however since we now construct our own regex from a pattern, we no longer need to preserve the `:`. This only is used for display purposes, and since we use {} everywhere, updating to do that